lib/internal/modules/esm/resolve.js

```diff
@@ -1026,6 +1026,35 @@ function throwIfInvalidParentURL(parentURL) {
   }
 }
 
+/**
+ * Process policy
+ */
+function processPolicy(specifier, context) {
```

Comment:
This needs JSDoc for the parameters.

Reply:
👍 Same as comment above, I purposely don't write those until I think the api is stable.